File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.java
import java.util.*;

class Solution {
    public String solution(String rsp) {
        StringBuilder sb = new StringBuilder();
        String[] arr = rsp.split("");
        String answer = "";
        
        String scissors = "2";
        String rock = "0";
        String paper = "5";
        
        for(String a:arr)
            if(a.equals(scissors))
                sb.append(rock);
            else if(a.equals(rock))
                sb.append(paper);
            else
                sb.append(scissors);
        
        answer = sb.toString();
        return answer;
    }
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    var answer = [];
    var _rsp = rsp.split("");
    
    for (var r of _rsp) {
        if (r == "2") {
            answer.push("0");
        } else if (r == "0") {
            answer.push("5");
        } else {
            answer.push("2");
        }
    }
    
    return answer.join("");
}